serverless/containers/reference-content/containers-concurrency.mdx

```diff
@@ -16,18 +16,68 @@ categories:
+                  │     ┌──────┐│               │    │     ┌──────┐│  
+                  │     │      ││               └────┼───► │      ││  
+3 requests  ──────┼───► │      ││   3 requests  ─────┼───► │      ││  
+                  │     │      ││               ┌────┼───► │      ││  
+                  │     └──────┘│               │    │     └──────┘│  
+                  │     ┌──────┐│                    └─────────────┘  
+                  │     │      ││                       1 service     
+            ┌─────┼───► │      ││                 1 container instance
+            │     │     │      ││                                     
+                  │     └──────┘│                                     
+                  └─────────────┘                                     
+                     1 service                                        
+               3 container instances                                  
+```
 
-When [deploying a container](/serverless/containers/how-to/deploy-container/), Scaleway Serverless Containers allows you to edit the **Maximum concurrent requests per instance** parameter.
+## Concurrency setting
 
-Serverless Containers apply the highest possible value by default: **80 concurrent requests**. We recommend using the default value, as Serverless Containers are designed to efficiently function with it, but you can lower it to better fit specific requirements.
+When [deploying a container](/serverless/containers/how-to/deploy-container/), Scaleway Serverless Containers allows you to configure the [concurrency](/serverless/containers/concepts/#concurrency), which is the maximum number of requests a single container instance can handle at the same time. The concurrency setting ranges from 1 to 1,000 simultaneous instances.
+
+By default, Serverless Containers allow **80 concurrent requests** per container instance. We recommend using the default value, as Serverless Containers are designed to efficiently function with it, but you can lower it to better fit specific requirements.
+
+## Impact of concurrency on container scaling
+
+If the number of incoming requests exceeds the set concurrency value for a container, Serverless Containers automatically creates new container instances to handle the additional load.
+
+For example, if a container is set to handle 10 concurrent requests, and 50 requests or events arrive at the same time, Serverless Containers will create at least 5 container instances to handle the incoming load.
+
+## Benefits of concurrency
+
+**Efficiency**: Setting a higher concurrency value allows a single container instance to handle more requests, which can reduce the total number of container instances, therefore reducing costs.
+
+**Responsiveness**: Setting a lower concurrency (down to 1 concurrent container instance) allows to reduce latency for individual requests, since the container focuses on fewer tasks at a time. However, 
```

Comment:
```suggestion
**Responsiveness**: Setting a lower concurrency (down to 1 concurrent container instance) allows to reduce latency for individual requests, since the container focuses on fewer tasks at a time.
```